contracts/src/utils/keccak256.cairo

```diff
@@ -437,22 +648,44 @@ mod tests {
     fn test_reverse_u64_word() {
         let array = array![0x12345656, 0x46e12df86, 0x23e098a8c5b850];
         let expected_result = array![0x5656341200000000, 0x86df126e04000000, 0x50b8c5a898e023];
-        assert_eq!(reverse_u64_word(array.span()), expected_result.span());
+        assert_eq!(reverse_u64_word(array.span(), 0), expected_result.span());
 
         let array = array![0x12345656, 0x46e12df86, 0x23e098a8c5b85040];
         let expected_result = array![0x5656341200000000, 0x86df126e04000000, 0x4050b8c5a898e023];
-        assert_eq!(reverse_u64_word(array.span()), expected_result.span());
+        assert_eq!(reverse_u64_word(array.span(), 0), expected_result.span());
 
         let array = array![
             0x007a9a2e1663480b, 0x3845df0d714e8caa, 0x49f9241e13a826a6, 0x78da3f366e546f2a
         ];
         let expected_result = array![
             0x0B4863162E9A7A00, 0xAA8C4E710DDF4538, 0xA626A8131E24F949, 0x2A6F546E363FDA78
         ];
-        assert_eq!(reverse_u64_word(array.span()), expected_result.span());
+        assert_eq!(reverse_u64_word(array.span(), 0), expected_result.span());
 
         let array = array![0x0165CC12];
         let expected_result = array![0x12cc6501];
-        assert_eq!(reverse_u64_word(array.span()), expected_result.span());
+        assert_eq!(reverse_u64_word(array.span(), 0), expected_result.span());
+
+        let array = array![
+            0x61a4bcca63b5e8a4,
+            0x6da3abe2080f75c1,
+            0x6c18467d5838f00b,
+            0x375d9ba4c7c313dd,
+            0x49d35915d0abec0a,
+            0x28990198bb32aa57,
+            0x0e681e7eb41a001c,
+            0x0094c7c36a712671
+        ];
+        let expected_result = array![
```

Comment:
how did we get that?

Reply:
it's just teh reverse endianness of the input span<u64>